### ISSUE TITLE
Blank pages during OCR:  "Error: The Page /Contents array contains an element which is not a stream"

### ISSUE THREAD

**Issue by AEgit** [jbarlow83/OCRmyPDF#398]
I tried to ocr the following PDF:
https://app.box.com/s/ffraogy4ayco5gc87t8kj406ww3o731v

using the following commands (after cd'ing into the working directory):
`for f in *.pdf; do ocrmypdf -v -l eng --force ./"$f" ./"${f%.pdf}_ocr.pdf"; done`

(I have tried both with and without the `--force` command)

A PDF with the same page number but containing only blank pages is produced.
During the OCR process, the following error messages are being displayed:

`**** Error: The Page /Contents array contains an element which is not a stream                                                      This page will not display correctly   `

Any thoughts?

**Comment by jbarlow83** [comment]
I can't reproduce it. My guess is it is a Ghostscript bug and upgrading to a new version will resolve the issue. The error comes form Ghostscript.

I have gs 9.26.

**Comment by AEgit** [comment]
Oh, I am so sorry, I uploaded the wrong file. This is the file that is causing the issue:
https://app.box.com/s/nqutypoot3l9wryjyvhtt91dgdv4iy8b

Software package versions employed:
ocrmypdf 8.3.0
tesseract 4.0.0-beta.1
gs 9.26
qpdf 8.0.2

**Comment by jbarlow83** [comment]
Can reproduce now, thanks.
What version of pikepdf is installed?

**Comment by AEgit** [comment]
According to `pip3 freeze` the following version of pikepdf is installed:
`pikepdf==1.4.0`

The operating system is Ubuntu 18.04 (64 bit) which is running as part of the Windows 10 Subsystem for Linux.

**Comment by jbarlow83** [comment]
>Windows 10 Subsystem for Linux.

I didn't know that worked! Interesting. I added a brief note to the documentation about this. Are there any caveats or does it amount to installing Ubuntu 18.04 and WSL?

**Comment by jbarlow83** [comment]
Fixed in v8.3.1

**Comment by AEgit** [comment]
Cheers, I can confirm that this has been fixed in v8.3.1.

Regarding the WSL: Yes, it is really straightforward - you just activate WSL and install Ubuntu 18.04 (I reckon it will work also with other distributions, but I have not tested that) and then install `ocrmypdf `as described in the documentation (I did it via `pip3`). This is also one of the reasons why I am not too much worried about a GUI for `ocrmypdf ` - as far as I know, you cannot use any GUIs at the moment with WSL. Note, that I have been using `ocrmypdf ` within WSL now for at least one year (if not longer) without any problems. So far, however, I have not tested the new WSL 2, which has just been released. I reckon there should be no problems with that as well, but I cannot guarantee.
